io/include/traccc/io/event_map2.hpp

```diff
@@ -52,13 +39,8 @@ struct event_map2 {
             // Candidate objects
             vecmem::vector<track_candidate> candidates;
 
-            for (const auto& meas_link : measurements) {
-
-                track_candidate cand = {
-                    detray::geometry::barcode{meas_link.surface_link},
-                    meas_link.meas};
-
-                candidates.push_back(cand);
+            for (const auto& meas : measurements) {
+                candidates.push_back(meas);
```

Comment:
Maybe unrelated to this PR, but should this get a call to ```reserve```?

Reply:
You are right. Fixed.